fcrepo-kernel-api/src/main/java/org/fcrepo/kernel/api/utils/iterators/RdfStream.java

```diff
@@ -359,6 +359,11 @@ public Triple apply(final T prototriple) {
         };
     }
 
+    protected static <From, To> Iterator<To> flatMap(final Iterator<From> i,
```

Comment:
`Iterators::transform` and `Iterators::filter` already exist.

1. `Iterable` implies that we can produce multiple iterators by calling `Iterable::iterator` several times. We don't want to make that claim.
2.  `Stream` is considerably heavier than `Iterator`. I'm very much in favor of `Stream` becoming more common in our code, but when we need only a very tight set of operations (and the majority of uses of `flatMap` are either alone or with a `filter` or two) it's not clear to me that it's worth the overhead in code or execution.

I'm not saying we shouldn't ever make `Stream`s at a low level, but I am saying that we shouldn't always make `Stream`s at a low level. Fundamentally, `Stream` represents a computation, but `Iterator` represents a flow of elements, and that's really what we have here.